fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1611,7 +1611,10 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	FlowLock fetchSemaphore;
 	int numFetchWaiters;
 	std::shared_ptr<ReadIteratorPool> readIterPool;
-	std::vector<Future<Void>> actors;
+	ThreadReturnPromiseStream<std::pair<std::string, double>> metricPromiseStream;
```

Comment:
Please create separate promise streams for each thread.

Reply:
Created